### ISSUE TITLE
Git workflow/etiquette

### ISSUE THREAD

**Issue by jmdejong** [floofytobles/procedural-text-adventure#1]
We should Decide on what we use for the git workflow.

My proposal:

The main development branch is `master`.
If we would ever reach a stable release we can make a separate branch for that (`release` or `1.0` or `stable`).

Don't push to `master`.
All development should happen on separate branches.
These branches can be merged into master using pull requests.
Don't merge a pull request immediately, wait until others have approved it (I think this is better as etiquette and not a hard rule).

Some good practices:
- Don't commit unneccessary files
- Make sure the commit message is descriptive
- Before you commit check the changes with `git status` and `git diff`

Force pushes can be a dangerous thing.
I propose to mostly use merge instead of rebase unless you're experienced with git already.

Open questions:
- Should we enforce a systematic naming scheme for bramches? For example to make all feature branches start with `f_`?
- Should we enforce some level of descriptiveness in commit messages?

What do you think about this?
